File: latihan b/java/TimSepakbola.java
public class TimSepakbola{

    //atribut
    private String nama;
    private String negara;
    private String tahun;
    private String pelatih;
    private String pemain;

    TimSepakbola(){
    }

    //prosedur dan fungsi getter setter
    public void setNamaTim(String nama){
        this.nama = nama;
    }

    public String getNamaTim(){
        return this.nama;
    }

    public void setNegara(String negara){
        this.negara = negara;
    }

    public String getNegara(){
        return this.negara;
    }

    public void setTahun(String tahun){
        this.tahun = tahun;
    }

    public String getTahun(){
        return this.tahun;
    }    

    public void setPelatih(String pelatih){
        this.pelatih = pelatih;
    }

    public String getPelatih(){
        return this.pelatih;
    }

    public void setPemain(String pemain){
        this.pemain = pemain;
    }

    public String getPemain(){
        return this.pemain;
    }
}

File: latihan b/c++/TimSepakbola.cpp
class TimSepakbola{
    private:

        //atribut private
        string namatim;
        string negara;
        string tahun;
        string pelatih;
        string pemain;


    public:

        TimSepakbola(){
        }

        //prosedur dan fungsi getter setter
        void setNamaTim(string namatim){
            this->namatim = namatim;
        }

        string getNamaTim(){
            return this->namatim;
        }

        void setNegara(string negara){
            this->negara = negara;
        }

        string getNegara(){
            return this->negara;
        }

        void setTahun(string tahun){
            this->tahun = tahun;
        }

        string getTahun(){
            return this->tahun;
        }

        void setPelatih(string pelatih){
            this->pelatih = pelatih;
        }

        string getPelatih(){
            return this->pelatih;
        }

        void setPemain(string pemain){
            this->pemain = pemain;
        }

        string getPemain(){
            return this->pemain;
        }

        ~TimSepakbola(){}
    

};